tasks/collect-data/collect-data.yaml

```diff
@@ -60,6 +60,15 @@ spec:
     - name: resultsDir
       type: string
       description: The relative path in the workspace to the results directory
+    - name: singleComponentMode
+      type: string
+      description: single component mode
+    - name: snapshotName
```

Comment:
Pretty unfortunate we have to do this since the EC task can't accept a namespaced name (which we already have)